MeDrop/MeDrop/Sources/Collection/View/Extension+CollectionView.swift

```diff
@@ -0,0 +1,25 @@
+//
+//  Extension+CollectionView.swift
+//  MeDrop
+//
+//  Created by JAESEOK LEE on 2023/07/11.
+//
+
+import Foundation
+import SwiftUI
+
+extension CollectionView {
+    func sortingButton(order: String) -> some View {
+        Button {
+            self.sortedBy = order
+        } label: {
+            Text(order)
+            Spacer()
+            if sortedBy == order {
+                Image(systemName: "checkmark.circle.fill")
+            } else {
+                Image(systemName: "checkmark.circle")
+            }
```

Comment:
삼항연산자로 바꾸면 알아보기 쉬울거같아요 ! ㅎㅎ

```suggestion
            Image(systemName: sortedBy == order ? "checkmark.circle" : "checkmark.circle.fill")
```

Reply:
감사합니다. 반영하겠습니다.